src/silx/meson.build

```diff
@@ -20,6 +11,20 @@ py.install_sources([
     'conftest.py',
     'py.typed',
 ],
-pure: false,    # Will be installed next to binaries
 subdir: 'silx',  # Folder relative to site-packages to install to
 )
+
+pure_subdirs = [
+  'app',
+  'gui',
+  'opencl',
+  'resources',
+  'sx',
+  'test',
+  'third_party',
+  'utils',
+]
+
+foreach subdir: pure_subdirs
+  install_subdir(subdir, install_dir: silx_dir)
+endforeach
```

Comment:
That's copied from numpy: https://github.com/numpy/numpy/blob/d936ca00751d55349d7304d67dfa9b40455e89b7/numpy/meson.build#L365-L370